doc/Maintenance_and_Troubleshooting.md

```diff
@@ -35,34 +35,77 @@ Another problem is that the CA might expire as well (normally after 10 years)
 
 ## Failed cluster deployment debugging
 
-Ask kubernetes what went wrong, e.g. `kubectl describe cluster <CLUSTER_NAME>`.
-The status and the events may give you a clue what happened. You can also look
-at the `openstackcluster` object.
-
-You can check at the openstack layer. A cluster deployment should result in a router,
-a network, a subnet, a loadbalancer (in front of kubeapi) and a number of servers (VMs)
-for the control-plane and worker nodes. Have you run out of quota?
+NOTE: The following `kubectl` and `clusterctl` commands should be executed against 
+the management Kubernetes cluster API. Keep in mind that these tools and the 
+`kubeconfig` to access the management Kubernetes cluster are available in the management
+host, hence it is convenient to execute the following commands from the management host.
+
+Ask Kubernetes what went wrong:
+```bash
+kubectl describe cluster <CLUSTER_NAME>
+```
+
+The status and the events may give you a clue what happened. The healthy cluster should
+be in the phase: `Provisioned`
+```bash
+$ kubectl describe cluster <CLUSTER_NAME> | yq .Status.Phase
+Provisioned
+```
+
+You can also have a look at the `openstackcluster` object and see OpenStack related
+statuses and events. The healthy cluster should be ready:
+```bash
+$ kubectl describe openstackcluster <CLUSTER_NAME> | yq .Status.Ready
```

Comment:
You can also do 
`kubectl get cluster <CLUSTER_NAME> -ojsonpath='{ .status.phase }'` and 
`kubectl get openstackcluster <CLUSTER_NAME> -ojsonpath='{ .status.ready }'` to avoid

Reply:
I just pushed this in.